src/command-line/users/add.js

```diff
@@ -11,7 +11,15 @@ program
 	.command("add <name>")
 	.description("Add a new user")
 	.on("--help", Utils.extraHelp)
-	.action(function (name) {
+	.option(
+		"--password [password]",
+		"new password, will be prompted if not specified on command line"
+	)
+	.option(
+		"--save-logs",
+		"if password is specified on command line, this will effect if logs for this user will be saved on disk"
```

Comment:
i think this should be "affect", not "effect"
but even so, i still don't understand what this option does

Reply:
`if password is specified on command line, this specified whether logs should be saved to disk (defaults to false)`

Reply:
only if a password is specified, this enables saving (a user's) logs to disk